tests/data_files/constant/bitstring-in-dn.pem

```diff
@@ -0,0 +1,51 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
For many of the moved files, the old copy in `data_files` is no longer used. I think we should remove those.

Reply:
Sure. 
But I do not think it is scope of this PR. I prefer cleanup it later.
I post some ideas in #7018. Those ideas haven't been break down, just record them in #7018

Reply:
Addressed in #7652

Reply:
Ok to do some of the cleanup later. But for the files that are moved, please change the git commit to do a move. If we have a commit that copies and another commit that deletes, it will be harder to follow the history.

Reply:
> Ok to do some of the cleanup later. But for the files that are moved, please change the git commit to do a move. If we have a commit that copies and another commit that deletes, it will be harder to follow the history.

The history has been re-written. And I have added some other parse function here.